src/lib/Libnet/net_server.c

```diff
@@ -223,20 +221,8 @@ conn_t *get_conn(int sock) {
  *
  */
 void
-connection_init(void) {
-	conn_t *cp = NULL;
-
-	if(!svr_allconns.ll_next) {
-		CLEAR_HEAD(svr_allconns);
-		return;
-	}
-
-	cp = (conn_t *)GET_NEXT(svr_allconns);
-	while(cp) {
-		int sock = cp->cn_sock;
-		cp = GET_NEXT(cp->cn_link);
-		close_conn(sock);
-	}
+connection_init(void) 
+{
```

Comment:
Might we leak socket descriptors due to the code that was removed?

Reply:
It should not leak here. We actually call connection_init only once at the start of the process, by which time there are no open connection handles, so the code to go over connection handles and closing them is redundant (probably vestigial). 

However, I see your point, that it might get used in ways it is not used today, so I will add that back.